package-lock.json

```diff
@@ -27070,6 +27071,11 @@
       "integrity": "sha1-8xwiIlqWMtK7+OSt2+8kCqdlph8=",
       "dev": true
     },
+    "node_modules/lodash.groupby": {
+      "version": "4.6.0",
+      "resolved": "https://registry.npmjs.org/lodash.groupby/-/lodash.groupby-4.6.0.tgz",
+      "integrity": "sha1-Cwih3PaDl8OXhVwyOXg4Mt90A9E="
+    },
```

Comment:
🤔  we don't use lodash anywhere else except in the `/stories-block` package. I'm curious about maybe moving the util that is in the dashboard (or copying it for the time being) that gives us the same `groupBy` functionality without importing lodash.  it's in `dashboard/src/utils/groupBy`

Reply:
Just for visibility:

The `groupBy` util in the dashboard doesn't do the same thing as lodash's but we also don't want to bring in lodash as a dep - so we opted to make our own.